packages/client/src/pages/GameItem/index.tsx

```diff
@@ -0,0 +1,195 @@
+            <div className={styles.gameInfoBlock}>
+              <div
+                className={styles.gameLogo}
+                style={{ backgroundImage: `url(${game?.logo})` }}
+              ></div>
+              <h2 className={styles.gameName}>{game?.name}</h2>
+              <div className={styles.gameInfoContent}>
+                <ul className={styles.gameInfoList}>
+                  <li className={styles.gameInfoItem}>
+                    <span className={styles.fieldName}>Developer</span>
+                    <span className={styles.fieldValue}>
+                      {game?.gameCreator.name}
+                    </span>
+                  </li>
+                  <li className={styles.gameInfoItem}>
+                    <span className={styles.fieldName}>Release date</span>
+                    <span className={styles.fieldValue}>
+                      {moment(game?.creationDate).format("DD-MM-YYYY")}
+                    </span>
+                  </li>
+                  <li className={styles.gameInfoItem}>
+                    <span className={styles.fieldName}>Genres</span>
+                    <span className={styles.fieldValue}>
+                      {game?.genres.map((genre, i) =>
+                        i ? `, ${genre.name}` : genre.name
+                      )}
+                    </span>
+                  </li>
+                  <li className={styles.gameInfoItem}>
+                    <span className={styles.fieldName}>Age rating</span>
+                    <span className={styles.fieldValue}>{game?.ageRating}</span>
+                  </li>
+                </ul>
+              </div>
+            </div>
+
+            <div className={styles.gameDescriptionBlock}>
+              <h2 className={styles.header}>About a game</h2>
+              <p className={styles.description}>{game?.description}</p>
+            </div>
+            <div className={styles.actionsBlock}>
+              <div className={styles.priceBlock}>
+                {game?.discount && (
+                  <>
+                    <span className={styles.saleSize}>
+                      {`-${game.discount.amount}%`}
+                    </span>
+                    <span className={styles.previousPrice}>{game.price} $</span>
+                  </>
+                )}
+
+                <span className={styles.currentPrice}>
+                  {Math.trunc(
+                    (game?.price as number) *
+                      (game?.discount ? (100 - game.discount.amount) / 100 : 1)
+                  )}
+                  $
+                </span>
+              </div>
+              <div className={styles.buttonsBlock}>
+                <button
+                  className={`${styles.actionBtn} ${
+                    game &&
+                    getUserSessionData().includes(game.id) &&
+                    styles.active
+                  }`}
+                  onClick={() => addToCartHandler()}
```

Comment:
How about `onClick={addToCartHandler}`? Your version also re-declares this callback on every render.